chrome/sunset.js

```diff
@@ -88,17 +82,17 @@ Sunset.maybeShowNotification_ = function() {
     // Trigger notification if we have reached the date suggested
     // by the timeline and if enough time has passed since the previous
     // one was shown. These checks are not required for the first notification.
-    var today = new Date();
+    var now = new Date();
```

Comment:
Nit: We can also have "today" if we setHours(0, 0, 0, 0), although it probably doesn't matter much if the timer goes off after midnight or later during the day.